lib/rucio/core/replica.py

```diff
@@ -4472,3 +4496,84 @@ def get_RSEcoverage_of_dataset(
             result[rse_id] = total
 
     return result
+
+
+@transactional_session
+def refresh_replicas(
+        rse_id: Optional[str] = None,
+        replicas: Optional['Iterable[dict[str, Any]]'] = None,
+        *,
+        session: "Session"
+) -> bool:
+    """
+    Updates the updated_at timestamp of the given replicas but don't wait if row is locked.
+
+    :param rse_id: the RSE containing the replicas to refresh.
+    :param replicas: a list with replicas (dictionary with the information of the affected replica).
+    :param session: The database session in use.
+
+    :returns: True, if successful, False otherwise.
```

Comment:
What returns `True` and `False` in this function feels a bit arbitrary - e.g. if the input is missing, `True` is returned, but it doesn't feel right since no refresh action has been taken.